server/index.ts

```diff
@@ -189,6 +193,69 @@ router.get('/:space/proposal/:id/votes', async (req, res) => {
     .then(obj => res.json(obj));
 });
 
+router.post('/:space/offchain_proofs', async (req, res) => {
+  const space = req.params.space;
+  const verifyingContract = req.body.verifyingContract;
+  const actionId = req.body.actionId;
+  const chainId = req.body.chainId;
+  const merkleRoot = req.body.merkleRoot;
+  const steps = req.body.steps;
+
+  if (
+    !space ||
+    !verifyingContract ||
+    !actionId ||
+    !chainId ||
+    !merkleRoot ||
+    !steps ||
+    steps.length < 1 // must have at least 2 steps
```

Comment:
Is the comment correct here? Maybe `// must have at least 1 step` ?

Reply:
ah, it should be < 2, because with only 1 step it is not valid. Fixed.